.idea/statesNames.py

```diff
@@ -0,0 +1,21 @@
+"""
+CP1404/CP5632 Practical
+State names in a dictionary
+File needs reformatting
+"""
+
+# TODO: Reformat this file so the dictionary code follows PEP 8 convention
+STATE_NAMES = {"QLD": "Queensland", "NSW": "New South Wales", "NT": "Northern Territory", "WA": "Western Australia",
+               "ACT": "Australian Capital Territory", "VIC": "Victoria", "TAS": "Tasmania"}
+# print(STATE_NAMES)
+
+state = input("Enter short state: ").upper()
+while state != "":
+    if state in STATE_NAMES:
+        print(state, "is", STATE_NAMES[state])
+    else:
+        print("Invalid short state")
+    state = input("Enter short state: ").upper()
+
+for name in STATE_NAMES:
+    print("{:.2s},  is {:4s}".format(name, STATE_NAMES[name]))
```

Comment:
By writing your string formatting as `("{:.2}".format(name))` you are trying to print a string that may exceed two characters, and limit it to only 2 character spaces. If you are trying to align this string to the left side, with varying spacing per `name` print, I propose using align in string formatting.

Doing this is quite simple, all you need to do is write your print as such
`print("{:<3}".format(name))`
By writing the your curly brace as `{:<3}` you are asking Python to leave up to 3 spaces after the start of the string. So if you use `name` and name equals `"QLD"`, it will count up to 3 spaces from the start of the string . If name equals `"NT"`, it will still do this, count 3 spaces from the start of the string, before it places whatever comes next in your print statement. If you print all the contents of your dictionary using this kind of formatting, they should align themselves quite nicely, and you won't need to write any contents of your second curly brace.

You can use other kinds of characters in align to achieve different results, such as `^` which will center the string, or `>`, which will count spaces behind the string.
If I didn't quite explain it well enough, you can find a more straight-forward explanation here, with examples: [https://pyformat.info/#string_pad_align](https://pyformat.info/#string_pad_align)